modules/backend/widgets/MediaManager.php

```diff
@@ -1290,72 +1290,81 @@ protected function getPlaceholderId($item)
      */
     protected function generateThumbnail($thumbnailInfo, $thumbnailParams = null)
     {
-        $tempFilePath = null;
-        $fullThumbnailPath = null;
-        $thumbnailPath = null;
         $markup = null;
 
-        try {
-            /*
-             * Get and validate input data
-             */
-            $path = $thumbnailInfo['path'];
-            $width = $thumbnailInfo['width'];
-            $height = $thumbnailInfo['height'];
-            $lastModified = $thumbnailInfo['lastModified'];
+        if($this->isVector($thumbnailInfo['path'])) {
+            $markup = $this->makePartial('thumbnail-image', [
+                'isError' => false,
+                'imageUrl' => Url::to(config('cms.storage.media.path') . $thumbnailInfo['path'])
+            ]);
+        }
+        else {
```

Comment:
@Rike-cz, could you fix styling? 😉 ([style guide](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md))

Why didn't you wrap it with `try-catch` block? `makePartial` - method could throw an exception and your temp.

Reply:
ok, I will repair this.